.vscode/launch.template.json

```
@@ -0,0 +1,59 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "type": "node",
+        "name": "Attach",
+        "port": 9229,
+        "request": "attach",
+        "skipFiles": ["<node_internals>/**"]
+      },
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Beacon",
+        "skipFiles": ["<node_internals>/**"],
+        "smartStep": true,
+        "program": "${workspaceFolder}/packages/cli/bin/lodestar.js",
+        "args": ["beacon"],
+        "console": "integratedTerminal"
+      },
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Dev",
+        "skipFiles": ["<node_internals>/**"],
+        "smartStep": true,
+        "program": "${workspaceFolder}/packages/cli/bin/lodestar.js",
+        "args": ["dev"],
+        "console": "integratedTerminal"
+      },
+      {
+        "name": "Test Current File",
+        "type": "node",
+        "request": "launch",
+        "program": "${workspaceFolder}/node_modules/.bin/vitest",
+        "args": ["--run", "${file}", "-t", "${input:testName}"],
```

Could we add a single thread flag here so the test starts up quicker?
```suggestion
        "args": [
            "--run",
            "--pool",
            "threads",
            "--poolOptions.threads.singleThread",
            "-t",
            "${input:testName}"
            "${file}"
        ],
```

---

this branch is out of date, I kinda wanna look into not requiring a vs code extension as well to make it easier to use

---

Now uptodate